test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/AzureFunctions/AzureFunctionsServiceTests.cs

```diff
@@ -21,8 +21,8 @@ class AzureFunctionsServiceTests
         public void AddSqlInputBinding()
         {
             // copy the original file because the input binding will be inserted into the file
-            string originalFile = Path.Join(testAzureFunctionsFolder, "AzureFunctionsNoBindings.ts");
-            string testFile = Path.Join(Path.GetTempPath(), string.Format("InsertSqlInputBinding-{0}.ts", DateTime.Now.ToString("yyyy-dd-M--HH-mm-ss")));
+            string originalFile = Path.Join(testAzureFunctionsFolder, "AzureFunctionsNoBindings.cs");
+            string testFile = Path.Join(Path.GetTempPath(), string.Format("InsertSqlInputBinding-{0}.cs", DateTime.Now.ToString("yyyy-dd-M--HH-mm-ss")));
```

Comment:
String interpolation is glorious!

Instead of
`string.Format("InsertSqlInputBinding-{0}.cs", DateTime.Now.ToString("yyyy-dd-M--HH-mm-ss"))`
just
`$"InsertSqlInputBinding-{DateTime.Now.ToString("yyyy-dd-M--HH-mm-ss")}.cs"`

Reply:
Also, not that it matters, but you probably want MM so that it spits out two-digit month numbers.